src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/ImportService.kt

```diff
@@ -61,6 +62,10 @@ class ImportService(
             movesTo: LocalDate,
             reportsTo: LocalDate): File? {
 
+        logger.info("Generating spreadsheet for supplier '$supplierName', moves from '$movesFrom', moves to '$movesTo' and reports to '$reportsTo'.")
+
+        if (movesFrom.plusMonths(2).isBefore(movesTo)) throw IllegalArgumentException("A maximum of two months data can be queried at a time.")
```

Comment:
@jcavell do we defo want this.  Bumped up to two months after the workshop call today.